doc/classes/PropertyTweener.xml

```diff
@@ -59,7 +78,25 @@
 
 				func tween_curve(v):
 				    return curve.sample_baked(v)
+				[/gdscript]
+				[csharp]
+				[Export]
+				public Curve Curve { get; set;}
+
+				public override void _Ready()
+				{
+				    Tween tween = CreateTween();
+				    // Interpolate the value using a custom curve.
+				    Tween.TweenProperty(this, "position:x", 300.0f, 1.0f).AsRelative().SetCustomInterpolator(TweenCurve);
```

Comment:
```suggestion
				    Callable tweenCurveCallable = Callable.From((float value) => TweenCurve(value));
				    tween.TweenProperty(this, "position:x", 300.0f, 1.0f).AsRelative().SetCustomInterpolator(tweenCurveCallable);
```
This one compiled and ran for me, though I'm not confident this is the preferred way to create a callable in this case. Needs a review from a C# expert.

(Used this page: [C# Callable](https://docs.godotengine.org/en/stable/tutorials/scripting/c_sharp/c_sharp_differences.html#callable))